plugins/b85/README.md

```diff
@@ -0,0 +1,26 @@
+# FoodsoftB85
+
```

Comment:
We recently decided about the plugin maintenance and agreed that non-default plugins should have a list of contact persons in the Readme file (See https://github.com/foodcoops/foodsoft/tree/master/plugins#plugin-maintenance)
Could you add a contact to this file whomever we/user can contact if the plugin does not work anymore as expected?